src/sentry/models/tagvalue.py

```diff
@@ -49,35 +47,6 @@ def is_valid_value(cls, value):
 
     def get_label(self):
         # HACK(dcramer): quick and dirty way to hack in better display states
-        if self.key == 'sentry:user':
-            return self.data.get('email') or self.value
-        elif self.key == 'sentry:function':
-            return '%s in %s' % (self.data['function'], self.data['filename'])
-        elif self.key == 'sentry:filename':
-            return self.data['filename']
-        elif self.key == 'sentry:release' and len(self.value) == 40:
+        if self.key == 'sentry:release' and len(self.value) == 40:
```

Comment:
Now that I see this value here I just noticed that this no longer reflects our up to date logic for making display values for releases. We should probably make this use the same method the UI and the release model uses.

Reply:
Can you give me a pointer to one of those?

Reply:
https://github.com/getsentry/sentry/blob/73fac98b9f6816c53ba1d60c0a810b05b0926795/src/sentry/models/release.py#L244-L252

Reply:
Thanks, fixed, you tell me if I Java'ed up the Python or not (I'm so rusty).